specification/domainservices/resource-manager/Microsoft.AAD/stable/2021-05-01/domainservices.json

```diff
@@ -0,0 +1,1113 @@
+          "description": "The continuation token for the next page of results.",
+          "readOnly": true
+        }
+      },
+      "description": "The response from the List Domain Services operation."
+    },
+    "DomainServiceProperties": {
+      "description": "Properties of the Domain Service.",
+      "properties": {
+        "version": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "description": "Data Model Version"
+        },
+        "tenantId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Azure Active Directory Tenant Id"
+        },
+        "domainName": {
+          "type": "string",
+          "description": "The name of the Azure domain that the user would like to deploy Domain Services to."
+        },
+        "deploymentId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Deployment Id"
+        },
+        "syncOwner": {
+          "readOnly": true,
+          "type": "string",
+          "description": "SyncOwner ReplicaSet Id"
+        },
+        "replicaSets": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ReplicaSet",
+            "description": "ReplicaSet"
+          },
+          "description": "List of ReplicaSets"
+        },
+        "ldapsSettings": {
+          "$ref": "#/definitions/LdapsSettings",
+          "description": "Secure LDAP Settings"
+        },
+        "resourceForestSettings": {
+          "$ref": "#/definitions/ResourceForestSettings",
+          "description": "Resource Forest Settings"
+        },
+        "domainSecuritySettings": {
+          "$ref": "#/definitions/DomainSecuritySettings",
+          "description": "DomainSecurity Settings"
+        },
+        "domainConfigurationType": {
+          "type": "string",
+          "description": "Domain Configuration Type"
+        },
+        "sku": {
+          "type": "string",
+          "description": "Sku Type"
+        },
+        "filteredSync": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "description": "Enabled or Disabled flag to turn on Group-based filtered sync",
+          "x-ms-enum": {
+            "name": "FilteredSync",
+            "modelAsString": true
+          }
+        },
+        "notificationSettings": {
+          "$ref": "#/definitions/NotificationSettings",
+          "description": "Notification Settings"
+        },
+        "migrationProperties": {
+          "readOnly": true,
+          "$ref": "#/definitions/MigrationProperties",
+          "description": "Migration Properties"
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "description": "the current deployment or provisioning state, which only appears in the response."
+        },
+        "configDiagnostics": {
```

Comment:
please mark this as readOnly (or mark individual properties within it as readOnly if the entirety is not)

Reply:
The entire property is no read-only